DataConnectors/ReadMe.md

```diff
@@ -180,7 +180,7 @@ Once you have a working POC, you are ready to build, validate the data connector
 3. **Validate the Connector UX** – Follow these steps to render and validate the connector UX you just built
     1.	The test utility can be accessed by this URL - https://aka.ms/sentineldataconnectorvalidateurl
     2.  Go to Microsoft Sentinel -> Data Connectors 
-    3.	Click the “import” button and select the json file you created as follows.
+    3.	Click the "import" button and select the json file you created as follows. To validate a codeless connector with this tool, be sure to only upload a json file containing the `connectorUiConfig` collection.
```

Comment:
Hi @austinmccollum Can you please update the text in accordance with the Azure documentation - [Create Codeless Connector](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/sentinel/create-codeless-connector.md), to ensure consistency, adding a screenshot for the same for your reference, thanks!
![image](https://user-images.githubusercontent.com/115772287/234562488-897e4ba9-2da8-4864-9556-9d80614475cd.png)

Reply:
Hi @austinmccollum thanks for letting us know, we will discuss with the team and update you shortly.